src/Okta.Idx.Sdk.E2ETests/Helpers/OktaSdkHelper.cs

```diff
@@ -34,6 +34,11 @@ public async Task<IUser> CreateActiveUserIdentifiedWithPasswordAsync(string emai
             return await _client.Users.CreateUserAsync(options);
         }
 
+        public Task<IUser> CreateUnassignedUserIdentifiedWithPasswordAsync(string userName, string password)
+        {
+            throw new NotImplementedException();
```

Comment:
I guess we're not using this method for 3.1 right?

Reply:
Yes, we don't need this for this feature